pkg/services/sqlstore/migrations/migrations.go

```diff
@@ -112,6 +112,10 @@ func (oss *OSSMigrations) AddMigration(mg *Migrator) {
 		accesscontrol.AddManagedDashboardAnnotationActionsMigration(mg)
 	}
 
+	if oss.features != nil && oss.features.IsEnabledGlobally(featuremgmt.FlagOnPremToCloudMigrations) {
```

Comment:
we usually don't want to have migrations behind a feature toggle
is there a reason for making an exception here?

Reply:
Only that we don't see the feature being ready for 11 so I thought it might be nice not to have the tables in 11. But we can remove the toggle.